### DISCUSSION TITLE
AG 1.1-Beta11 Potential Buffer Issue in LPX

### DISCUSSION THREAD

**Discussion by dsponaugle** [apohl79/audiogridder#319] [Help]
Thanks for the ongoing development of this amazing program! I removed and Beta10 and installed Beta11 but I'm experiencing an issue and unsure if it's LPX or AG. Basically I've got 2 Windows VMs running as servers and my host mac running the plugin. I've got the 2 servers set to different ID's but when I try to adjust the buffer for Server ID 1, it also adjusts the buffer for another AG plugin instance on a different track with AG set to the Server ID 2. In other words, I can't independently adjust the buffers anymore even though they are separate servers on separate LPX tracks

Things I've tried: 

-Uninstalling all AG instances from all 3 machines
-Reset LPX preferences to default and eventually deleted the LPX preference file
-Tried an empty LPX Project rather than the template I was using prior

Is there a definitive guide to completely removing AG from my machines via regedit on the windows site and a deeper uninstall on the Mac? I had this functionality before but not sure if this issue lies within LPX or AG. I also remember seeing a greyed out section in AG server settings that mentioned the ID was being sent via command-line or something of that nature. Regardless it showed the correct ID in the settings.

**Comment by apohl79** [comment]
The buffer settings are shared across all plugin instances at the moment. No individual settings by server possible.

**Reply by dsponaugle** [reply]
Did we have the ability to change buffer settings per server before or has this always been the way? I just recently started using a second server and thought I had this functionality in 10 but could have been mistaken. I like to have a server set to 0ms for Guitar amp sims and whatnot but don't require that for most other scenarios when near-realtime isn't necessary. Thanks again for everything!

**Reply by dsponaugle** [reply]
I just reverted the VMs to a snapshot with Beta10 and noticed the same functionality. Thanks for clearing it up!

**Comment by apohl79** [comment]
Yea it was always like that. I never considered this use case as the audio latency is defined by the “slowest” insert.
